BaseFilter/sections/foreign.txt

```diff
@@ -2019,6 +2019,8 @@ ylilauta.org#%#//scriptlet('abort-on-stack-trace', 'Element.prototype.append', '
 ||static.ylilauta.org/js/*/Module/FrameLoader.js$replace=/this\.#?(contentLoadFailed|showFail)\(\);//
 ||ylilauta.org^$document,replace=/(<script src="https:\/\/static\.ylilauta\.org\/js\/.+?\/Module\/FrameLoader\.js" )integrity=".*?" (type="module"><\/script>)/\$1\$2/
 ||ylilauta.org^$removeheader=link
+ylilauta.org#%#(function(){const b=window.alert;window.alert=function(a){if(!/you are using an ad blocker/.test(a.toString()))return b(a)};})();
+ylilauta.org##.toast-root
```

Comment:
Have you checked these rules with the app?
Seems they are not needed on my end.
I'd limit them only for extensions.

Reply:
I meant this directive

```
!#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android)
```